### ISSUE TITLE
Add Stop Running Code Option

### ISSUE THREAD

**Issue by kaylinland** [sgsinclair/Voyant#514]
Would it be possible to add a "Stop" button for code cells?

**Comment by ajmacdonald** [comment]
I assume this has to do with cancelling a long-running process, but can you please expand on what you mean by this?

**Comment by kaylinland** [comment]
Yes--basically, if you run a cell that produces an error message, the message pops up and after you acknowledge the error, a pop-up message still reads "running" and it seems the cell is still working. I'm not sure if this affects running the other cells.
